packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -25,6 +25,7 @@ class WelcomePage extends StatelessWidget {
     return SmoothScaffold(
       backgroundColor: backgroundColor,
       brightness: Brightness.dark,
+      resizeToAvoidBottomInset: false,
```

Comment:
This wasn't supposed to be changed, It has nothing to do with #3177

Reply:
Here also  exists bottom overflow error ! when selecting country . You can checkout the page before changes